src/integrations/watchers/indexable-author-watcher.php

```diff
@@ -72,6 +72,7 @@ public function delete_indexable( $user_id ) {
 		}
 
 		$indexable->delete();
+		\do_action( 'wpseo_indexable_deleted', $indexable );
```

Comment:
Do we have prominent words related to authors?

Reply:
No, but we decided to leave this do_action in case that in the future we do have them and then we don't need to think about this :)